cypress/integration/dib-travel/company-employees/employees-user.spec.ts

```diff
@@ -30,14 +30,15 @@ describe('Company Employees - Employees (User)', () => {
   it('should allow user to add new employee with sending invitation to employee', () => {
     cy.waitForAngular();
 
-    addEmployee(employee.firstName, employee.lastName, employee.email, true);
+    addEmployee(employee.firstName, employee.lastName, employee.email);
 
     cy.get('dib-people-management dib-employees dib-page .grid .name-cell').should('contain', employee.firstName);
     cy.get('dib-people-management dib-employees dib-page .grid .name-cell').should('contain', employee.lastName);
     cy.get('dib-people-management dib-employees dib-page .grid .table-cell').should('contain', employee.email);
   });
 
   it('should allow user to search a certain employee', () => {
+    //TODO revisit selector one more time
```

Comment:
I don't know if this comment was left by mistake, but I'm not sure it can be better written. This work also:` dib-people-management dib-employees dib-page ui-input .input`

Reply:
Another possibility: `dib-people-management dib-employees dib-page ui-input .placeholder`